API.md

```diff
@@ -191,11 +152,46 @@ Example Response:
 }
 ~~~
 
-### Batch Create (proposed)
+### Create
+
+| Patterns | Payloads | Responses
+| ---     | ---     | ---
+| `/create.action` | `{JSON}` | 201 `Location: http://devstore.rerum.io/v1/id/11111` `{JSON}`
+
+- **`{JSON}`**—The object to create
+- **Response: `{JSON}`**—Containing various bits of information about the create.
+
+Add a completely new object to RERUM and receive the location URI
+in response.  Accepts only single JSON objects for RERUM storage.
+Mints a new URI and returns the object's `Location` as a header.
+If the object already contains an `@id` that matches an object in RERUM,
+the API will direct the user to use [update](#update) instead.
+
+Example Response:
+
+- **Header:** `Location: http://devstore.rerum.io/v1/id/11111`
+- **Body:**
+
+~~~ (json)
+{
+  "code" : 201,
+  "new_obj_state" : {
+    "@id": "http://devstore.rerum.io/v1/id/11111",
+    ...
+  },
+  "iiif_validation" : {

```

Comment:
We should take this out, but it is there for now.

Reply:
Yeah, that was a pretty big change that I didn't think was in scope of this pass at it.  Like you said, that is probably part of another mini sprint to "Make it more like an API".